### ISSUE TITLE
Chrono dependency involves insecure version

### ISSUE THREAD

**Issue by itamarst** [cardoe/stderrlog-rs#31]
```
   = ID: RUSTSEC-2020-0071
   = Advisory: https://rustsec.org/advisories/RUSTSEC-2020-0071
   = ### Impact
     
     Unix-like operating systems may segfault due to dereferencing a dangling pointer in specific circumstances. This requires an environment variable to be set in a different thread than the affected functions. This may occur without the user's knowledge, notably in a third-party library.
     
     The affected functions from time 0.2.7 through 0.2.22 are:
     
     - `time::UtcOffset::local_offset_at`
     - `time::UtcOffset::try_local_offset_at`
     - `time::UtcOffset::current_local_offset`
     - `time::UtcOffset::try_current_local_offset`
     - `time::OffsetDateTime::now_local`
     - `time::OffsetDateTime::try_now_local`
     
     The affected functions in time 0.1 (all versions) are:
     
     - `at`
     - `at_utc`
     - `now`
     
     Non-Unix targets (including Windows and wasm) are unaffected.
     
     ### Patches
     
     Pending a proper fix, the internal method that determines the local offset has been modifan `Err` on the `try_*` methods and `UTC` on the non-`try_*` methods.
     
     Users and library authors with time in their dependency tree should perform `cargo update     
     Users of time 0.1 do not have a patch and should upgrade to an unaffected version: time 0     
     ### Workarounds
     
     No workarounds are known.
     
     ### References
     
     time-rs/time#293
   = Announcement: https://github.com/time-rs/time/issues/293
   = Solution: Upgrade to >=0.2.23
   = time v0.1.43
     └── chrono v0.4.19
         └── stderrlog v0.5.1
             └── fil4prod v0.1.0
```

I have personally solved this by switching dependency from `chrono` to `time` directly, since the latter basically sufficed and `chrono` does not appear to be getting updated in near term (https://github.com/chronotope/chrono/pull/578).

**Comment by cardoe** [comment]
So the issue here is that time doesn't do the same formatting support that chrono has. Time will only go to second resolution and nothing below that. I personally depend on more accurate resolution than second in my code. Any suggestions to a solution are welcome.

**Comment by virtualritz** [comment]
The [gist in the resp. issue on `chrono`]( https://github.com/chronotope/chrono/pull/578#issuecomment-1075907329) is that it is unaffected by `RUSTSEC-2020-0071`.

**Comment by cardoe** [comment]
It is unfortunately vulnerable to https://rustsec.org/advisories/RUSTSEC-2020-0159 but not the issue this ticket was originally opened about. So we can move forward on this.

**Comment by cardoe** [comment]
@edmorley contributed the ability to disable timestamps via a feature flag in #45 that's been included in 0.5.3. So you'll be able to disable this to avoid chrono, so I'll close this since there's no version of chrono out that fixes this. The time crate doesn't have feature parity without breaking existing behavior so there's currently no solution.
